index.js

```diff
@@ -4,7 +4,47 @@
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
  * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
  */
-function calculator() {}
+
+let i;
+let result = 0;
+//const parameters = process.argv;
+function calculator() {
+  
+
+// this "IF" is there is no parameters
+ if (process.argv.length == 2 ) { 
+     return "No operation provided..."
+ }
+
+ //
+  if (process.argv[2] === "modulo"){
+        
+    return "Invalid operation: modulo"
```

Comment:
`modulo` is hardcoded here. Another invalid operator could be `division`, however, the function does not return the error message.